frontend/src/metabase/public/components/LogoBadge.jsx

```diff
@@ -13,7 +13,7 @@ const LogoBadge = ({ dark }) => (
     <LogoIcon height={28} dark={dark} />
     <span className="text-small">
       <span className="ml1 md-ml2 text-medium">{jt`Powered by ${(
-        <span className={dark ? "text-white" : "text-brand"}>
+        <span key="metabase" className={dark ? "text-white" : "text-brand"}>
```

Comment:
Got curious what's the use of this particular key.

Was there a React warning for it?

Reply:
Yeah, `jt` i18n helper seem to create an array of React elements internally, so when we render JSX inside `jt`, we need to pass a `key`. Otherwise we're getting a regular warning about missing prop